src/java/Main.java

```diff
@@ -0,0 +1,33 @@
+public class Main {
+    public static void main(String[] args) {
+        ElectricPokemon pikachu = new ElectricPokemon("Pikachu",
+                1, 35, "kaas", "pikachu", "thunder punch", "4");
+        pikachu.eats();
+        pikachu.speaks();
+        pikachu.thunderPunch();
+        pikachu.electroBall();
+
+        GrassPokemon bulbasaur = new GrassPokemon("Bulbasaur", 1, 45,
+                "seed", "bulbasaur", "leech Seed", "1");
+        bulbasaur.eats();
+        bulbasaur.speaks();
+        bulbasaur.leechSeed();
+        bulbasaur.solarBeam();
+
+        WaterPokemon evolutions = new WaterPokemon("Evolutions", 1, 79,
+                "water", "evolution", "Hydro Pump", 1);
+        evolutions.eats();
+        evolutions.speaks();
+        evolutions.hydroPump();
+        evolutions.surf();
+
+        FirePokemon charmander = new FirePokemon("Charmander", 1, 39,
+                "fire", "charmander", "pyroBall", 1);
+
```

Comment:
Dit is meer een tip, probeer goed te kijken of je gebruik wil maken van witregels (regels zonder code) om je code leesbaarder te maken. En wanneer je dit wel en niet wilt doen. Dit i.v.m. dat afwijkingen opvallen, deze regel heb je bijvoorbeeld niet gebruikt na regel 11 en 18